src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --main-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, `font-family: Roboto, Arial, sans-serif;` ensures that if 'Roboto' is not available, 'Arial' will be used instead.